index.html

```diff
@@ -10,7 +10,145 @@
+                fill="#6070FF" />
+        </svg>
+
+        <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path fill-rule="evenodd" clip-rule="evenodd"
+                d="M3 7C3 6.44772 3.44772 6 4 6H20C20.5523 6 21 6.44772 21 7C21 7.55228 20.5523 8 20 8H4C3.44772 8 3 7.55228 3 7ZM3 12C3 11.4477 3.44772 11 4 11L20 11C20.5523 11 21 11.4477 21 12C21 12.5523 20.5523 13 20 13L4 13C3.44772 13 3 12.5523 3 12ZM4 16C3.44772 16 3 16.4477 3 17C3 17.5523 3.44772 18 4 18H20C20.5523 18 21 17.5523 21 17C21 16.4477 20.5523 16 20 16H4Z"
+                fill="#6070FF" />
+        </svg>
+
+    </header>
+    <section class="headline-main">
+        <section class="headline">
```

Comment:
- [ ] Kindly reduce the `margin-top` of the headline to match the design in Figma 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/212054537-fd61cb6c-d871-429f-807f-5e00c626e585.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/212054567-9528f172-1a21-468d-b920-cfcca60605d5.png)